lib/serializer/JsonDeserializer.h

```diff
@@ -37,8 +37,8 @@ class DLL_LINKAGE JsonDeserializer: public JsonTreeSerializer<const JsonNode *>
 	void serializeInternal(int64_t & value) override;
 
 private:
-	void readLICPart(const JsonNode & part, const TDecoder & decoder, const bool val, std::vector<bool> & value);
-	void readLICPart(const JsonNode & part, const TDecoder & decoder, std::set<si32> & value);
+	static void readLICPart(const JsonNode & part, const TDecoder & decoder, const bool val, std::vector<bool> & value);
```

Comment:
Same here. I think better to remove this "modernizing" rule entirely

Reply:
About LIC - it needed to move out of classes, because it is just duplicated as is.

Reply:
Done as new interface.